event-handler/build.gradle

```
@@ -51,11 +51,14 @@ dependencies {
     api project(':shared')
     implementation "androidx.annotation:annotation:$annotations_ver"
     implementation "androidx.work:work-runtime:$work_runtime"
+    // Base64
+    implementation "org.apache.commons:commons-collections4:4.1"
```

can we use this library for base64 
```suggestion
    implementation "commons-codec:commons-codec:1.15"
```
As this is the direct dependency we want to use also we are facing dependency issue with currently used library in testapp. 
I did the change and tested it with testapp and it's working fine.